scripts/tests/test_import_open_textbook_library.py

```diff
@@ -0,0 +1,213 @@
+                        "title": None,
+                        "first_name": "Lorraine",
+                        "middle_name": None,
+                        "last_name": "Melita",
+                        "location": None,
+                        "background_text": "Lorraine Melita",
+                    },
+                    {
+                        "id": 5249,
+                        "contribution": "Author",
+                        "primary": False,
+                        "corporate": False,
+                        "title": None,
+                        "first_name": "Patricia",
+                        "middle_name": None,
+                        "last_name": "Roiger",
+                        "location": None,
+                        "background_text": "Patricia Roiger",
+                    },
+                ],
+                "subjects": [
+                    {
+                        "id": 57,
+                        "name": "Early Childhood",
+                        "parent_subject_id": 5,
+                        "call_number": "LB1139.2",
+                        "visible_textbooks_count": 11,
+                        "url": "https://open.umn.edu/opentextbooks/subjects/early-childhood",
+                    }
+                ],
+                "publishers": [
+                    {
+                        "id": 874,
+                        "name": "Open SUNY",
+                        "url": "https://textbooks.opensuny.org",
+                        "year": 2020,
+                        "created_at": "2020-07-21T23:48:48.000Z",
+                        "updated_at": "2020-07-21T23:48:48.000Z",
+                    }
+                ],
+            },
+            {
+                "identifiers": {"open_textbook_library": "895"},
+                "source_records": ["open_textbook_library:895"],
+                "title": "The ELC: An Early Childhood Learning Community at Work",
+                "languages": ["eng"],
+                "description": "The ELC professional development model was designed to improve the quality of teacher candidates",
+                "subjects": ["Early Childhood"],
+                "publishers": ["Open SUNY"],
+                "authors": [
+                    {"name": "Heather Bridge"},
+                    {"name": "Lorraine Melita"},
+                    {"name": "Patricia Roiger"},
+                ],
+                "lc_classifications": ["LB1139.2"],
+            },
+        ),
+        # Test case 3: None values
+        (
+            {
+                'id': 730,
+                'title': 'Mythology Unbound: An Online Textbook for Classical Mythology',
+                'ISBN10': None,
+                'ISBN13': None,
+                'language': None,
+                'contributors': [
+                    {
+                        'first_name': None,
+                        'last_name': None,
+                        'contribution': None,
```

Comment:
Is this a valid case?

Reply:
Ah, if this is not a case we will ever encounter, then that might not be the best test! The new example seems fine.